addon-resizer/nanny/main/pod_nanny.go

```diff
@@ -111,10 +113,55 @@ func main() {
 		BaseMemory:    *baseMemory,
 		MemoryPerNode: *memoryPerNode,
 	}
-	nannycfg, err := loadNannyConfiguration(*configDir, nannyConfigurationFromFlags)
+
+	period := time.Duration(*pollPeriod) * time.Millisecond
+	hc := healthcheck.NewHealthCheck(*hcAddress, period*5)
+	hc.Serve()
+
+	n := &nanny.Nanny{
+		StopChan:       make(chan struct{}, 1),
+		ScaleDownDelay: *scaleDownDelay,
+		ScaleUpDelay:   *scaleUpDelay,
+		PollPeriod:     period,
+		HealthCheck:    hc,
+		Threshold:      uint64(*threshold),
+		Client:         k8s,
+	}
+
+	runNanny(*n, *nannyConfigurationFromFlags, *configDir)
```

Comment:
nannyConfigurationFromFlags and configDir doesn't change, can we have them in the Nanny struct?

Reply:
Updated!